android/build.gradle

```diff
@@ -1,8 +1,15 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
+apply plugin: 'com.android.library'
+apply from: 'gradle-maven-push.gradle'
+
 buildscript {
   repositories {
-    jcenter()
+      mavenLocal()
+      jcenter()
+      maven {
+          url "$projectDir/../../react-native/android"
```

Comment:
I believe this should be `url "$projectDir/../node_modules/react-native/android"`, right? same on line 26